packages/documentation/copy/en/project-config/Compiler Options.md

```diff
@@ -54,75 +54,66 @@ tsc app.ts util.ts --target esnext --outfile index.js
 <p>Show all compiler options.</p>
 </td></tr>
 
-<tr class='even' name='generateTrace'>
-  <td><code>--generateTrace</code></td>
-  <td><p><code>string</code></p>
-</td>
-</tr>
-<tr class="option-description even"><td colspan="3">
-<p>Generates an event trace and a list of types.</p>
-</td></tr>
-
-<tr class='odd' name='help'>
+<tr class='even' name='help'>
   <td><code>--help</code></td>
   <td><p><code>boolean</code></p>
 </td>
 </tr>
-<tr class="option-description odd"><td colspan="3">
+<tr class="option-description even"><td colspan="3">
 <p>Gives local information for help on the CLI.</p>
 </td></tr>
 
-<tr class='even' name='init'>
+<tr class='odd' name='init'>
   <td><code>--init</code></td>
   <td><p><code>boolean</code></p>
 </td>
 </tr>
-<tr class="option-description even"><td colspan="3">
+<tr class="option-description odd"><td colspan="3">
 <p>Initializes a TypeScript project and creates a tsconfig.json file.</p>
 </td></tr>
 
-<tr class='odd' name='listFilesOnly'>
+<tr class='even' name='listFilesOnly'>
```

Comment:
Not in this PR but one day https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child